src/main/java/seedu/duke/template/LrTemplate.java

```diff
@@ -10,7 +10,7 @@
                                             + "\t|             |\n"
                                             + "\t+----+V_ac+---+\n";
 
-    private Inductor inductor;
+    private final Inductor inductor;
```

Comment:
Here as well.

Reply:
Since the way we set values is calling `Component.setValue()`, these actually still work when they are `final`.

Reply:
Yup it will work, but the point of declaring it as final is sort of nullified

Reply:
In a sense, we _are_ using the same `LoadComponent` object, just changing its value.

Reply:
Fair, then might have to make the names capitalized right